### PR TITLE
Adding myself to the AUTHORS file

### PR DESCRIPTION
**Pull request by annag42**
Trying again, from my fork on GitHub, to see what it looks like when it fails again (now that my signature and email align and are turned on).

### CODE REVIEW

**Comment by annag42** [comment]
TL;DR: Closing, to re-try from non-main branch on local machine. ​

Longer version:
Closing this PR, too, which does indeed seem to be failing identically despite the verified email for the signed commit, which supports the model of the failure being caused by submitting from a fork rather than directly from my local machine. 

Further, the change in this PR was committed directly to main, while the custom for this project is to commit changes to a separate (new, temporary) branch, then submit a pull request to create a new branch in the shared project. 
To follow this customary procedure, I plan to take these steps:
1. Revert the change to main locally on my machine.
2. Create a new branch locally on my machine.
3. Re-do the change locally, on the new branch.
4. Commit the change locally.
5. Push the local change to this shared project, which should create a pull request.
6. Confirm status, and make adjustments as necessary
